Dockerfile.test

```diff
@@ -0,0 +1,133 @@
+FROM camunda/camunda-bpm-platform:7.9.0 as base
+
+SHELL ["/bin/bash", "-c"]
+
+USER root
+
+RUN rm -rf /camunda/webapps/camunda-invoice
+RUN rm -rf /camunda/webapps/examples
+RUN apk add wget curl busybox-extras -f
+
+USER camunda
+
+COPY --chown=camunda:camunda ./camunda.sh /camunda/
+COPY context.xml /camunda/conf/
+
+FROM base as build
+
+USER root
+
+RUN apk add maven openjdk8 -f
+
+COPY pom.xml /dependencies/
+ADD "https://www.random.org/cgi-bin/randbyte?nbytes=10&format=h" skipcache
+
+RUN cd /dependencies/ && \
```

Comment:
Actually, you don't need these files in the testing image.

Just build library itself (you can remove further steps with `debian` and `camunda-bpm-platform` images) and copy here `test.sh` script which will run tests.
You don't even need to build demo processes here or seed.